aea/cli/common.py

```diff
@@ -488,7 +510,6 @@ def _copy_package_directory(
     # copy the item package into the agent's supported packages.
     item_type_plural = item_type + "s"
     src = str(package_path.absolute())
-    dest = os.path.join(ctx.cwd, "vendor", author_name, item_type_plural, item_name)
     logger.debug("Copying {} modules. src={} dst={}".format(item_type, src, dest))
```

Comment:
Aren't you missing removal of dest from clean_paths here?

Reply:
No. It's added to a clean_paths in `_add_item` and removed in decorator.